### PR TITLE
WFLY-13464 Change server startup documentation to reference local installation and standalone-microprofile.xml

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-13464

### CODE REVIEW

**Comment by emmartins** [comment]
1) We don't need the wildfly maven plugin config in the pom.xml anymore right?
2) It would still be preferable to include the start standalone section instead, for instance it includes a note for Windows users

**Comment by rhusar** [comment]
@emmartins Re no. 1, no, it's still needed for wildfly:deploy target (see https://github.com/wildfly/quickstart/pull/416/files#diff-d3218658e824c230fa8fcedb01baa886R38).

**Comment by emmartins** [comment]
@rhusar I mean that doesn't need the problematic version config in the plugin right?

File: microprofile-openapi/README.adoc [hunk 1]
```diff
@@ -20,28 +20,15 @@ To complete this guide, you will need:
...
-$ mvn wildfly:execute-commands -Dwildfly.commands=/extension=org.wildfly.extension.microprofile.openapi-smallrye:add
-$ mvn wildfly:execute-commands -Dwildfly.commands=/subsystem=microprofile-openapi-smallrye:add
-----
-
-. Since the MicroProfile OpenAPI subsystem modifies {productName}'s deployment chain, the server needs to be reloaded.
-+
-[source, shell]
-----
-$ mvn wildfly:shutdown -Dwildfly.reload=true
+$ {jbossHomeName}/bin/standalone.sh --server-config=standalone-microprofile.xml &
```
Comment (rhusar) [inline microprofile-openapi/README.adoc:31]: You can cherrypick the first commit from https://github.com/wildfly/quickstart/pull/423 and use 

```
:standalone-server-type: microprofile
...
include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
```

Comment (rhusar) [reply microprofile-openapi/README.adoc:31]: Update: no need to cherry-pick, already merged.

Comment (emmartins) [reply microprofile-openapi/README.adoc:31]: @pferraro can you please go for @rhusar suggestion, aligning with all other MP QS after updates?

File: microprofile-openapi/README.adoc [hunk 2]
```diff
@@ -20,28 +20,15 @@ To complete this guide, you will need:
...
 ----
 
 . Build and deploy the sample application to {productName}.
```
Comment (emmartins) [inline microprofile-openapi/README.adoc:34]: instead this should be:

// Build and Deploy the Quickstart
include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]

and after the content wrt using/accessing the app:

// Undeploy the Quickstart
include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]

File: microprofile-openapi/README.adoc (hunk 1 above)
Comment (pferraro) [reply microprofile-openapi/README.adoc]: Done.

File: microprofile-openapi/README.adoc (hunk 2 above)
Comment (pferraro) [reply microprofile-openapi/README.adoc]: Done.

**Comment by emmartins** [comment]
looks good to me, thanks @pferraro

**Comment by emmartins** [comment]
ready to merge @pferraro ?

**Pull request merged**